docs/quickstart-gke.md

```diff
@@ -233,20 +239,12 @@ Re-run the `kubeprod install` command, from the [Deploy BKPR](#step-2-deploy-bkp
   gcloud iam service-accounts delete ${GCLOUD_SERVICE_ACCOUNT}
   ```
 
-### Step 4: Delete the GKE cluster
+### Step 5: Delete the GKE cluster
 
   ```bash
   gcloud container clusters delete ${GCLOUD_K8S_CLUSTER}
   ```
 
-### Step 5: Delete any leftover GCE disks
```

Comment:
Since we've added the step that waits for the kubeprod namespace to be deleted. This step is not required (from experience)